SygicUIKit.podspec

```diff
@@ -33,11 +33,10 @@ TODO: Add long description of the pod here.
   s.resources = 'SygicUIKit/Resources/**/*.{xib,svg,jpg,png}'
   s.frameworks = 'UIKit'
   s.resource_bundle = {
-	'SygicUIKit' => ['SygicUIKit/Resources/**/*.{ttf}']
+	'SygicUIKit' => ['SygicUIKit/Resources/**/*.{ttf}'],
+    'SygicUIKitStrings' => ['SygicUIKit/Resources/Langs/*.lproj/*.strings']
```

Comment:
Neviem ci to len tu tak vyzera ale nejaky divny spacing to hadze, hore je tab a dole 4 medzeri